tcms/telemetry/views.py

```diff
@@ -1,6 +1,7 @@
 from django.views.generic import TemplateView
 
 
+# pylint: disable=missing-permission-required
 class TestingBreakdownView(TemplateView):  # pylint: disable=missing-permission-required
```

Comment:
Same comment as above.

Reply:
This file still includes other class/function level disable comments so all of them need to be fixed.